File: src/my_project_typescript/easy_problems_typescript/from51To100/determineColorChessboardSquare.ts
function squareIsWhite(coordinates: string): boolean {

    return (coordinates[0].charCodeAt(0) + coordinates[1].charCodeAt(0))%2 !==0
};


console.log(squareIsWhite("a1"))

console.log(squareIsWhite("h3"))

console.log(squareIsWhite("c7"))

File: src/my_project/easy_problems/from151To200/determine_color_chessboard_square.py
from typing import List 

class Solution:
    def squareIsWhite(self, coordinates: str) -> bool:



        return (ord(coordinates[0]) + ord(coordinates[1]))%2 != 0


solution = Solution()

print(solution.squareIsWhite(coordinates = "a1"))

print(solution.squareIsWhite(coordinates = "h3"))

print(solution.squareIsWhite(coordinates = "c7"))
